src/ViewHelpers/StudentApplicationFormViewHelper.php

```diff
@@ -82,23 +82,23 @@ protected static function displayPageFormThree(): string
      */
     protected static function displayPageFormFour(array $data): string
     {
-        $output = '<div class="row"><label>Select start date(s)</label><ul class="startDatesList">';
+        $output = '<div id="cohorts" class="row"><label>Select start date(s)</label><ul class="startDatesList">';
         foreach ($data['cohorts'] as $cohorts) {
-            $output .= '<li><label><input type="checkbox" data-nextcourse="false" ';
-            $output .= 'class="startDatesCheckbox" name="startDatesCheckbox"';
+            $output .= '<li><label class="cohort_checkbox"><input name="cohort" type="checkbox" data-nextcourse="false" ';
+            $output .= 'class="startDatesCheckbox cohort_checkbox submitApplicant"';
             $output .= ' value="' . $cohorts['id'] . '"/>';
             $output .= date_format(date_create_from_format("Y-m-d", $cohorts['date']), "D j M Y");
             $output .= '</label></li>';
         }
         $output .= '<li><label><input type="checkbox" data-nextcourse="true" ';
-        $output .= 'class="startDatesCheckbox" name="startDatesCheckbox"';
+        $output .= 'class="startDatesCheckbox submitApplicant" name="startDatesCheckbox"';
         $output .= ' value="register interest">next available online course (register interest)';
         $output .= '</label></li></ul>';
-        $output .= '<div id="startDateError" class="alert hidden formItem_alert"></div>';
+        $output .= '<div id="cohortError" class="alert hidden formItem_alert"></div>';
```

Comment:
Semantics  - cohortError mightn't be as clear as 'startDateError' if they're both referring to the same thing

Reply:
We had to change this to Cohort in order for the logic to work in saveApplicant.js, line 20